src/js/shield_defs.js

```diff
@@ -3164,6 +3164,53 @@ export function loadShields() {
     textColor: Color.shields.white,
   };
 
+  // Peru
+  shields["PE:national"] = {
+    spriteBlank: ["shield_pe_2", "shield_pe_3"],
+    textColor: Color.shields.black,
+    padding: {
+      left: 3,
+      right: 3,
+      top: 7,
+      bottom: 4,
+    },
+  };
+  [
+    "AM",
+    "AN",
+    "AP",
+    "AR",
+    "AY",
+    "CA",
+    "CU",
+    "HU",
+    "HV",
+    "IC",
+    "JU",
+    "LA",
+    "LI",
+    "LM",
+    "LO",
+    "MD",
+    "MO",
+    "PA",
+    "PI",
+    "PU",
+    "SM",
+    "TA",
+    "TU",
+    "UC",
+  ].forEach(
+    (department) =>
+      ([
+        shields[`PE:departmental:${department}`],
+        shields[`PE:rural:${department}`],
+      ] = [
+        escutcheonDownShield(12, Color.shields.white, Color.shields.black),
```

Comment:
These shields are visually closer to the state highway shields of Mexico (see line 584). Let's factor that out into a reusable shield definition and use it here.

Side note: this is making it really clear how much we need a draw function for badge shields, and maybe support for crossbars and accent colors in some of the functions...